ex03.ex

```diff
@@ -55,7 +55,13 @@ defmodule Ex03 do
 
   """
 
-  def odd_even . . . "your code"
+  def odd_even([]), do: []
```

Comment:
## -1

You could take this a step further with:

~~~ elixir
  def odd_even([]), do: []
  def odd_even([ h | t ]), do: [ atom_for(h) | odd_even(t) ]

  defp atom_for(n) when Integer.is_odd(n),  do:  :odd
  defp atom_for(n) when Integer.is_even(n), do:  :even
~~~